tests/unit/downloader/test_xls_generation.py

```diff
@@ -0,0 +1,165 @@
+import json
+import unittest
+
+from openpyxl import Workbook
+from openpyxl.worksheet.worksheet import Worksheet
+
+from ingest.downloader.downloader import XlsDownloader
+
+
+class XLSGenerationTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.downloader = XlsDownloader()
+        self.workbook = Workbook()
+        self.workbook.create_sheet(title='Project')
+
+    def test_given_input_has_only_1_set_of_data_successfully_creates_a_workbook(self):
+        #given
+        project_sheet_title = 'Project'
+        input_json = {
```

Comment:
data prep should be in its own function.

Reply:
I prefer to have it like this as everything is together

Reply:
OK, I moved it